README.md

```diff
@@ -26,6 +26,14 @@ See [`docs/Ruleset.md`](docs/Ruleset.md) for an exhaustive list of currently sup
 
 Verification codes are a mean to uniquely identify a semantic versioning trigger (a condition which upon detection, requires your code changes to be versioned).
 
+## Install
+
+You can install it globally:
```

Comment:
I would not suggest installing this tool globally as it does depend on stuff which may cause conflict with other tools if required globally.